### ISSUE TITLE
[Accessibility: ACC-255.1] Icon: Provide alternative text for image.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12589]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Add an appropriate alt attribute for images including poster images of videos that do not contain alt attributes. An alt attribute should be a concise and meaningful replacement for the image. For example, when describing a picture of a red balloon, entering alt="red balloon" will be more helpful than alt="This is a picture of a red balloon". The alt text of the image should be in the language of the page or if the page contains multiple languages the language of the surrounding or any referring content.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Card](https://sap.github.io/fundamental-ngx/#/core/card)
All FD-AVATAR (role=img) elements do not have a mechanism that allows an accessible name value to be calculated.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/0a03d16a-0cd7-4270-9b1d-358ccbb55214">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

The avatars in UI5 have an API to set aria-labelledby or aria-describedby from outside/app level: https://ui5.sap.com/#/api/sap.m.Avatar%23associations

I assume you will need this also on your side (you can start with labelling as a start).

Best Regards,
Nikolay

**Comment by InnaAtanasova** [comment]
@divyamalhotra488 Avatar has an input property called `ariaLabel` that sets aria-label and alt attributes and can be set by the user. Let us know if this doesn't work for you or you need something more. Thanks!

<img width="1635" height="865" alt="Image" src="https://github.com/user-attachments/assets/8dd48e75-9577-4369-a488-9c8001981cad" />
